src/openms/include/OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/CoarseIsotopeDistribution.h

```diff
@@ -0,0 +1,305 @@
+// --------------------------------------------------------------------------
+//                   OpenMS -- Open-Source Mass Spectrometry
+// --------------------------------------------------------------------------
+// Copyright The OpenMS Team -- Eberhard Karls University Tuebingen,
+// ETH Zurich, and Freie Universitaet Berlin 2002-2017.
+//
+// This software is released under a three-clause BSD license:
+//  * Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of any author or any participating institution
+//    may be used to endorse or promote products derived from this software
+//    without specific prior written permission.
+// For a full list of authors, refer to the file AUTHORS.
+// --------------------------------------------------------------------------
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Chris Bielow $
+// $Authors: Clemens Groepl, Andreas Bertsch, Chris Bielow $
+// --------------------------------------------------------------------------
+
+#ifndef OPENMS_CHEMISTRY_ISOTOPEDISTRIBUTION_COARSEID_H
+#define OPENMS_CHEMISTRY_ISOTOPEDISTRIBUTION_COARSEID_H
+
+#include <OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/IsotopePatternGenerator.h>
+
+
+namespace OpenMS
+{
+
+  /**
+    *         @ingroup Chemistry
+    *         @brief Isotope pattern generator for coarse (nominal mass) isotope distributions.
```

Comment:
can you explain what coarse really means? I dont think I understand the term here, also I am not sure whether "nominal mass" is used correctly here, e.g. see Wikipedia:

> The nominal mass for an element is the mass number of its most abundant naturally occurring stable isotope

which would imply that you only take the most abundant isotope for each ion, which is not what you are doing here?

Reply:
you are right, the description is incorrect

Reply:
What would you prefer?

Reply:
What would you prefer?

Reply:
http://mass-spec.lsu.edu/msterms/index.php/Mass_number
mass number, A

Sum of the numbers of protons and neutrons in an atom, molecule, or ion.

    Note: When expressed as a mass in daltons, mass number is equivalent to nominal mass.

Hmm I guess that is because some people also use "nominal mass" in this context.